### PR TITLE
Welcome section updates

### PR DESCRIPTION
**Pull request by Kuboosh21**
* Added messages
* centered on small screens
* gave more breathing room
* made title bigger on small screens to stay interesting
* added non-breaking spaces to welcome descriptions

Medium screen 
image

Small screen (600-900px wide)
image

XS screen (<600px)
image

### CODE REVIEW

File: web/src/components/home/subcomponents/welcome/messages.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const messages = {
+  discover: "Discover your rhythm at",
+  title: "Do Your Own Swing",
+  classesAndSocial:
+    "West Coast Swing classes and social dancing in San\u00A0Jose,\u00A0CA.",
+  consciousSpace:
+    "A COVID-Conscious, Queer-Led, 2SLGBTQIA+ Affirming\u00A0Space",
```
Comment (jesseez) [inline web/src/components/home/subcomponents/welcome/messages.js:7]: Try using `text-wrap: balance` instead

Also please add the rainbow icon at the end of affirming space

**Review by jesseez — APPROVED** [review]
lgtm

File: web/src/components/home/subcomponents/welcome/messages.js (hunk 1 above)
Comment (Kuboosh21) [reply web/src/components/home/subcomponents/welcome/messages.js]: Now it sort of breaks only in 1 spot, which may look unbalanced on specific screen sizes, but we should be okay

File: web/src/components/home/subcomponents/welcome/messages.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const messages = {
+  discover: "Discover your rhythm\u00A0at",
```
Comment (jesseez) [inline web/src/components/home/subcomponents/welcome/messages.js:2]: Could use balance for this one too

**Review by jesseez — APPROVED** [review]

**Pull request merged**